postsView.js

```diff
@@ -0,0 +1,64 @@
+const Post = require("./post");
+
+class PostsView{
+  constructor(postsModel, api){
+    this.loggedInContainer = document.querySelector("#logged-in");
+    this.postContainer = document.querySelector("#post-container");
+    this.postsModel = postsModel;
+    this.api = api;
+    this.user_id = null;
+    this.session_key = null;
+
+    document.querySelector('#submit-login').addEventListener('click', () => {
+      const user = document.querySelector('#input-username');
+      const password = document.querySelector('#input-password');
+      let userVal = user.value;
+      this.api.newSession(userVal, password.value, (data) => {
```

Comment:
Is there a reason for this inconsistency? You use two different ways to access the value